src/kolibri_android/application.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from android.activity import register_activity_lifecycle_callbacks
+
+
+class BaseActivity(object):
```

Comment:
Nice. We don't have 2 classes inheriting it but the abstraction looks ready for upstreaming or adding a second activity.